tests/server/test_init.py

```diff
@@ -0,0 +1,66 @@
+from unittest import mock
+
+import entrypoints
+import pytest
+
+from mlflow.exceptions import MlflowException
+from mlflow import server
+
+
+@pytest.fixture()
+def mock__exec_cmd():
```

Comment:
```suggestion
def mock_exec_cmd():
```

We can use `_` instead of `__`.

Reply:
Can you apply the same fix to other functions?